ci-operator/step-registry/cloud-bulldozer/OWNERS

```diff
@@ -1,4 +1,4 @@
-pprovers:
+approvers:
```

Comment:
here, instead of needing to edit each of the files you can use a symbolic link to have this main one and then reference it in each of the subfolders. This way we wont have to constantly edit the owners in multiple places or if there are issues like this

Reply:
Yes, makes sense when we have same group of owners for the entire repo. Updated as suggested.